content/references/http-websocket-apis/public-api-methods/ledger-methods/ledger.md

```diff
@@ -191,27 +107,28 @@ The response follows the [standard format][], with a successful result containin
+| `ledger.ledger_index`          | String  | The [Ledger Index][] of this ledger, as a quoted integer. |
+| `ledger.parent_close_time`     | Number  | The time at which the previous ledger was closed. |
+| `ledger.parent_hash`           | String  | The unique identifying hash of the ledger that came immediately before this one, as hexadecimal. |
 | `ledger.total_coins`           | String  | Total number of XRP drops in the network, as a quoted integer. (This decreases as transaction costs destroy XRP.) |
-| `ledger.transaction_hash`      | String  | Hash of the transaction information included in this ledger, as hex |
-| `ledger.transactions`          | Array   | (Omitted unless requested) Transactions applied in this ledger version. By default, members are the transactions' identifying [Hash][] strings. If the request specified `expand` as true, members are full representations of the transactions instead, in either JSON or binary depending on whether the request specified `binary` as true. |
-| `ledger_hash`                  | String  | Unique identifying hash of the entire ledger. |
+| `ledger.transaction_hash`      | String  | [Hash](basic-data-types.html#hashes) of the transaction information included in this ledger. |
+| `ledger.transactions`          | Array   | _(Omitted unless requested)_ Transactions applied in this ledger version. By default, members are the transactions' identifying [Hash][] strings. If the request specified `expand` as true, members are full representations of the transactions instead, in either JSON or binary depending on whether the request specified `binary` as true. |
+| `ledger_hash`                  | String  | The unique identifying hash of the entire ledger, as hexadecimal. |
 | `ledger_index`                 | Number  | The [Ledger Index][] of this ledger. |
 | `validated`                    | Boolean | _(May be omitted)_ If `true`, this is a validated ledger version. If omitted or set to `false`, this ledger's data is not final. |
-| `queue_data`                   | Array   | _(Omitted unless requested with the `queue` parameter)_ Array of objects describing queued transactions, in the same order as the queue. If the request specified `expand` as true, members contain full representations of the transactions, in either JSON or binary depending on whether the request specified `binary` as true. Added by the [FeeEscalation amendment][]. [New in: rippled 0.70.0][] |
+| `queue_data`                   | Array   | _(Omitted unless requested with the `queue` parameter)_ Array of objects describing queued transactions, in the same order as the queue. If the request specified `expand` as true, members contain full representations of the transactions, in either JSON or binary depending on whether the request specified `binary` as true. |
+
+The `ledger.accountState` field (omitted unless requested with `"full": true` or `"accounts": true`) is deprecated.
```

Comment:
```suggestion
The `ledger.accountState` field is deprecated.
```

^^ I feel this would convey the meaning better, since `full` and `accounts` have been deprecated themselves

Reply:
I felt the note was needed because otherwise it might be confusing to be told that a field you don't see is deprecated—people might wonder if it has been removed already. This makes it clear when you should expect to see the field even though it is deprecated.